src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
- [ ] On your `readme.md`, please, kindly implement the [Microverse Readme Template](https://github.com/microverseinc/readme-template), instead of the getting started CRA readme.  

In almost all the Microverse projects, this is a general requirement.  All sections are required except for the "tests" and "live demo" if not required by the project.

Reply:
Readme file updated